Be/unit1 wrap up

beccaelenzil opened the pull request:
the description of the activity is super short, but I'm not sure what more is needed. Thanks for any suggestions!

tildeee commented on config.yaml:
```diff
@@ -522,3 +522,14 @@ Standards:
         Path: /python-oop/problem-set-static-methods-and-class-methods.md
+  -
+    Title: Unit 1 Wrap-up
+    Description: "Reflection Activity"
```
Could this be "Activity: Mind Map," to match the file name, instead?

```suggestion
    Description: "Activity: Mind Map"
```

tildeee commented on unit1-wrap-up/activity-mind-map.md:
```diff
@@ -0,0 +1,13 @@
+# Unit 1 - Reflection Activity
```
Could we conform this to:

```suggestion
# Activity: Mind Map
```

? I'm really hoping for file names, lesson names, what's on the config, and what shows up on Learn, to all be aligned (so that we don't lose files as easily as we did in textbook-curriculum)

beccaelenzil replied:
incorporated.

tildeee commented on unit1-wrap-up/activity-mind-map.md:
```diff
@@ -0,0 +1,13 @@
+# Unit 1 - Reflection Activity
+
+Wow! We've learned so much in the last 5 weeks. Let's take time to reflect on our learning and make connections.
```
Because it'd be great to not tie our curriculum to time passing, could we divorce it from "five weeks"?

```suggestion
Wow! We've learned so much in this unit. Let's take time to reflect on our learning and make connections.
```

tildeee commented on unit1-wrap-up/activity-mind-map.md:
```diff
@@ -0,0 +1,13 @@
+# Unit 1 - Reflection Activity
+
```
Nitpicking: I know this is goofy, but I'm suggesting a template header.

```suggestion

## Goals

```

beccaelenzil replied:
incorporated.

tildeee commented on unit1-wrap-up/activity-mind-map.md:
```diff
@@ -0,0 +1,13 @@
+# Unit 1 - Reflection Activity
+
+Wow! We've learned so much in the last 5 weeks. Let's take time to reflect on our learning and make connections.
+
```
Nitpicking: Adding more of the headers from the activity template

```suggestion

Activity Instructions

```

tildeee commented on unit1-wrap-up/activity-mind-map.md:
```diff
@@ -0,0 +1,13 @@
+
+Wow! We've learned so much in the last 5 weeks. Let's take time to reflect on our learning and make connections.
+
+We will use a technique called a mind map. A mind map is a diagram that can represent tasks, ideas, concepts, etc. and the connections between them. In our case, we can include anything from our learnings from Unit 1 -- whether it be python programming concepts, study skills, learnings about how to be in community, or anything else.
+
+These are a few tools our team recommends: 
+* [Google Slide](https://www.google.com/slides/about/)
+* [Mind Meister](https://www.mindmeister.com/)
+
+and here is a [more exhaustive list](https://zapier.com/blog/best-mind-mapping-software/) of mind mapping software.
+
+We encourage you to pick a tool and go for it -- no censorship needed.
```
Recommending for this be reworded so the "we" is about us learners, without the instructor/team divide.

Also recommending "we will" -> "let's" since it's about to happen.

- Capitalization suggestions
- Framing the directions as activity instructions instead of as suggestions
```suggestion
Let's create a mind map. A mind map is a diagram that can represent tasks, ideas, and concepts, and the connections between them.

In our case, let's include any topic from our learnings from this unit. We should consider:

- Python programming concepts
- Study skills
- Learnings about how to be in community
- Anything else!

We can make our mind map using any of these tools:

- [Google Slide](https://www.google.com/slides/about/)
- [Mind Meister](https://www.mindmeister.com/)
- [Software from this list of mind mapping software](https://zapier.com/blog/best-mind-mapping-software/)
```

beccaelenzil replied:
I didn't see this before I started making changes, so I incorporated it, but didn't straight up commit the suggestion.

tildeee commented on unit1-wrap-up/activity-mind-map.md:
```diff
@@ -0,0 +1,13 @@
+* [Google Slide](https://www.google.com/slides/about/)
+* [Mind Meister](https://www.mindmeister.com/)
+
+and here is a [more exhaustive list](https://zapier.com/blog/best-mind-mapping-software/) of mind mapping software.
```
Why not just pick a few from this list? A number of these aren't free

beccaelenzil replied:
I tried to pick some, and I didn't love anything

tildeee reviewed (changes requested):
A lot of my suggestions were conforming it to the Activity template ~ Sorry not sorry haha.

Overall, as a student, from these instructions, I think I would have these questions:

- Am I doing this individually or in a group?
- What do these look like? Are there examples?
- What does a good mind map look like?
- What should I be getting out of this so I know I'm doing a good job?
- How long should I spend on this? Can I be done after ten minutes? Should I spend 3 hours on it?
- How do I know I'm finished?
- Do I just keep this on my computer? Do I ever share this with someone?

As an instructor, I'd have the questions:

- Did this need to be an in-class activity? Could this have been individual homework?
- When do I know a student is finished?
- What do I tell a student if they're having trouble getting started?
- How can I tell if my students are engaged with this, or getting something out of it?
- What will students do with this? Is there a submission?
- Are all the students just hanging out in one Zoom call?
- Will it be silent? Will there be conversation?

I'll let you decide how you want to address those questions, but personally I would probably put:

- An example section with two or three examples
- A time recommendation
- A reason for why the students are doing this in roundtable

beccaelenzil commented:
I reworked. Simon, I didn't see your suggested changes before I started reworking, but I think I incorporated them all. I can't believe I didn't include examples.

One question that's lingering for me is, should this really be an online activity? Should we give students the option to hand draw these or collaborate with classmates on the computer?

tildeee commented on unit1-wrap-up/activity-mind-map.md:
```diff
@@ -0,0 +1,41 @@
+- [Mind Meister](https://www.mindmeister.com/)
+- [Software from this list of mind mapping software](https://zapier.com/blog/best-mind-mapping-software/)
+
+We will spend approximately 40 minutes in small groups to work together and include all the wisdom from our classmates. We will come back together as a large group to share insights.
```
Can we change the language here to:

```suggestion
Spend approximately 40 minutes in small groups to work together and include all the wisdom from our classmates. 

Then, come back together as a large group to share insights.
```


Personally, I think that having example questions that the groups should share with the large group might be helpful. (For example, once it gets to large group sharing, the groups might be like "uhhh what do we share?"). For example, will it be a presentation, or just an open discussion?

If it's a presentation, I might include presentation directions (each group will present for 5 minutes each) and maybe:

```
When other groups are presenting, ask yourselves:

- What topics did this group and your group have in common?
- When comparing similar topics, what connections were similar?
- What connections were different?
- What can you learn from them?
```

If it's just an open discussion, I'd propose giving them the first opening question,

"We'll start the large group discussion with each group answering the question: ..."

tildeee commented on unit1-wrap-up/temp.py:
```diff
@@ -0,0 +1,11 @@
+def calculator( nums, operation ): 
+    num1 = nums[0] 
+    num2 = nums[1] 
+    if operation == '+':
+        total = num1 + num2 
+    elif operation == '-' 
+        ..... 
+    
+    return total
+
+calculator([2,3], '+'):
```
I think this file was included on accident.

tildeee reviewed (approved):
There's something in here that makes me go, "Hmmmm if I were a student, I'd wish that there was a concrete direction that said:

1. Something that says that students will be listing concepts and then drawing arrows inbetween them (not just "a diagram that _can_ represent something")

"Boxes are single concepts. Arrows and lines are made between concepts to show that they are connected somehow."

and

2. I wish there was something that said that the students should concretely be contributing to the diagram.

"Take turns contributing to the diagram."



Otherwise looks good! You added in an extra file.
